ts/features/bonus/bpd/screens/iban/insertion/IbanInsertionComponent.tsx

```diff
@@ -48,35 +56,46 @@ export const IbanInsertionComponent: React.FunctionComponent<Props> = props => {
       contextualHelp={props.contextualHelp}
     >
       <SafeAreaView style={IOStyles.flex}>
-        <View style={[IOStyles.horizontalContentPadding, IOStyles.flex]}>
-          <View spacer={true} large={true} />
-          <H1>{title}</H1>
-          <View spacer={true} large={true} />
-          <Body>{body1}</Body>
-          <View spacer={true} large={true} />
-          <H5>{ibanDescription}</H5>
-          <Item error={isInvalidIban}>
-            <Input
-              value={iban}
-              maxLength={IbanMaxLength}
-              onChangeText={text => setIban(text.toUpperCase().trim())}
-            />
-          </Item>
-          <View spacer={true} large={true} />
-          <View spacer={true} small={true} />
-          <Body>{body2}</Body>
-        </View>
-
-        {withKeyboard(
-          <FooterTwoButtons
-            rightDisabled={!userCanContinue}
-            onRight={() => Iban.decode(iban).map(props.onIbanConfirm)}
-            onCancel={props.onContinue}
-            rightText={I18n.t("global.buttons.continue")}
-            leftText={props.cancelText}
-          />,
-          true
-        )}
+        <ScrollView>
+          <View style={[IOStyles.horizontalContentPadding, IOStyles.flex]}>
+            <View spacer={true} large={true} />
+            <H1>{title}</H1>
+            <View spacer={true} large={true} />
+            <Body>
+              {body1}
+              <H4>{body1Bold}</H4>
+            </Body>
+            <View spacer={true} large={true} />
+            <H5>{ibanDescription}</H5>
+            <Item error={isInvalidIban}>
+              <Input
+                value={iban}
+                autoCapitalize={"characters"}
+                maxLength={IbanMaxLength}
+                onChangeText={text => {
+                  // On Android we cannot modify the input text, or the text is duplicated
+                  if (Platform.OS === "android") {
```

Comment:
The PR lgtm, as an improvement/suggestion we may use here `Platform.select`
```suggestion
                  Platform.select({
                    android: setIban(text),
                    default: setIban(text.toUpperCase().trim())
                  })}
```